backend/src/news/utils.py

```diff
@@ -1,12 +1,12 @@
-from urllib.parse import quote
-import requests
 import json
 from sqlalchemy import delete, insert, select
 from sqlalchemy.orm import Session
-from bs4 import BeautifulSoup
 from src.models import user_news_association_table, NewsArticle
 from openai import OpenAI
 from src.auth.database import SessionLocal
+from src.crawler.udn_crawler import UDNCrawler
+
+crawler = UDNCrawler()
 
 def store_news(news_data):
```

Comment:
It seems like this function only pass the received `news_data` to `crawler.save(news_data)` without additional process. How about remove this function and just directly use `crawler.save(...)` instead?